bindings/rust/wasmedge-sys/src/vm.rs

```diff
@@ -17,39 +15,67 @@ pub struct Vm {
 }
 
 impl Vm {
-    pub fn create(config: &crate::config::Config) -> Self {
-        let ctx = unsafe {
-            wasmedge::WasmEdge_VMCreate(config.ctx, std::ptr::null_mut() /* store */)
+    pub fn create(config: Option<&Config>, store: Option<&Store>) -> Option<Self> {
```

Comment:
same here

Reply:
Accepted. Thanks!

Reply:
May we use `result` return type for this?
Thanks.